joplin/api/schema.py

```diff
@@ -417,6 +418,13 @@ def resolve_page_type(self, info):
     def resolve_janis_urls(self, info):
         return self.janis_urls()
 
+    def resolve_events(self, info):
```

Comment:
it looks like this is being copied around a bit, we could probably reduce copypasta by using [graphene.Interface](https://docs.graphene-python.org/en/latest/types/interfaces/) like we're doing with [DepartmentResolver](https://github.com/cityofaustin/joplin/blob/745bc1fd95cebe90c76acba9018017673cae9876/joplin/api/schema.py#L302)

Reply:
thank you! i updated it to use graphene.Interface